server/src/serializers/ReviewSerializer.js

```diff
@@ -0,0 +1,21 @@
+class ReviewSerializer {
+  static summaryForShow = (reviews) => {
+    const allowedReviewFields = [
+      "id",
+      "createdAt",
+      "body",
+      "starRating",
+      "title",
+      "userId"]
```

Comment:
make your `]` go on its own line below if you are multilining this...
actually, this should be all on one line, its not more than 80 characters